src/thread_private_spike_store.hpp

```diff
@@ -36,7 +36,7 @@ public :
             spikes.insert(spikes.begin(), b.begin(), b.end());
         }
 
-        return spikes;
+        return std::move(spikes);
```

Comment:
Revert this change.

When returning by value, the return value will be used as an rvalue at the calling site. Though in this case NRVO will lead to copy elision.